### PR TITLE
Issue 38: Remove Dead Scripts

### PR DESCRIPTION
**Pull request by lucidmachine**
Remove:
* Fuck Shit Up script
* Standup tasks and scripts
* HTTP Status Code Lookup script
* Plugins:
** Google Translate
** Pug Me

### CODE REVIEW

File: lib/initializers/scheduler.ls [hunk 1]
```diff
@@ -8,9 +8,5 @@ module.exports = (robot) !->
 
   if process.env.DEBUG?
     schedule '*/10 * * * * *', \check-for-upcoming-events
-    # schedule '*/10 * * * * *', \prompt-standup
-    # schedule '*/13 * * * * *', \remind-about-standup
   else
     schedule '0 0 9 * * *', \check-for-upcoming-events
-    # schedule '0 0 8 * * *', \prompt-standup
-    # schedule '0 0 12 * * *', \remind-about-standup
```
Comment (egillespie) [inline lib/initializers/scheduler.ls]: Do you feel like taking on converting this file to plain JavaScript?

File: package.json [hunk 2]
```diff
@@ -17,7 +17,6 @@
   "dependencies": {
     "@firebase/app": "^0.4.14",
     "@firebase/firestore": "^1.4.9",
-    "cheerio": "^0.22.0",
```
Comment (egillespie) [inline package.json]: Surprise dependency! 😂

Don't forget to add your name to the `contributors` section of this file!

File: lib/initializers/scheduler.ls (hunk 1 above)
Comment (lucidmachine) [reply lib/initializers/scheduler.ls]: I've got my eyes on #37 in the near-future, but I didn't want to convert anything just yet.

File: package.json (hunk 2 above)
Comment (lucidmachine) [reply package.json]: Contributed!

File: lib/initializers/scheduler.ls (hunk 1 above)
Comment (egillespie) [reply lib/initializers/scheduler.ls]: 👍

File: package.json [hunk 3]
```diff
@@ -11,7 +11,8 @@
     "Erik Gillespie <erik.gillespie@gmail.com>",
     "James Green",
     "Joe Ryan",
-    "Patrick Hawkins"
+    "Patrick Hawkins",
+    "Eric Slenk <lucid.machine@gmail.com>"
```
Comment (egillespie) [inline package.json]: Awesome! ✨

**Comment by egillespie** [comment]
Merging to `master` and I'll deploy the changes after Hacktoberfest. Thanks for cleaning house! 🤘

**Pull request merged**